plugins/rbac/README.md

```diff
@@ -21,14 +21,28 @@ Follow the RBAC backend plugin [README](https://github.com/janus-idp/backstage-p
 
 **NOTE**
 
-To enable create role button on Administration -> RBAC roles list page, the role associated with your user should have the following permission policies associated with it. Add the following in your permission policies configuration file:
+- To enable create role button on Administration -> RBAC roles list page, the role associated with your user should have the following permission policies associated with it. Add the following in your permission policies configuration file:
 
 ```CSV
-p, role:default/team_a, catalog-entity, read, allow
+p, role:default/team_a, catalog-entity, read, allow  # only needed for users with the role other than `role:default/rbac_admin`
 p, role:default/team_a, policy-entity, create, allow
 g, user:default/<login-id/user-name>, role:default/team_a
 ```
 
+- To view the permissions from other plugins like `Kubernetes` and `Jenkins` as mentioned [here](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/rbac-backend/docs/permissions.md), add the following configuration in your `app-config.yaml`:
+
+```yaml title="app-config.yaml"
+permission:
+  enabled: true
+  rbac:
+    pluginsWithPermission:
+      - kubernetes
+      - jenkins
```

Comment:
What happens if the user adds this in the config explicitly?

Reply:
@invincibleJai  Sharing some screenshots here: 

<img width="954" alt="Screenshot 2024-02-02 at 12 38 08 PM" src="https://github.com/janus-idp/backstage-plugins/assets/22490998/1e5a09f8-0271-4de2-90ce-ff4e726fb3ed">
<img width="954" alt="Screenshot 2024-02-02 at 12 38 59 PM" src="https://github.com/janus-idp/backstage-plugins/assets/22490998/fa552fff-58e0-40d3-9910-263fb14346dc">